internal/common/common.go

```diff
@@ -113,3 +115,34 @@ func AllStrings(vs []string, f func(string) bool) bool {
 	}
 	return true
 }
+
+func Execute(command string, args ...string) (stdout string, stderr string, exitCode int) {
```

Comment:
why not under `pkg`? and why not an interface?

Reply:
It's a utility. Whoever uses it (for example `release.go`) can wrap it inside an interface.

Reply:
Added as an interface. Why should it be under `pkg`? It's a utility.

Reply:
because it's generic and probably can be used by other components (repos) it's not something that related directly to the service.